ModelVale/ModelVale/CustomComponents/HealthBarView.h

```diff
@@ -0,0 +1,16 @@
+//
+//  HealthBar.h
+//  ModelVale
+//
+//  Created by Chaytan Inman on 7/14/22.
+//
+
```

Comment:
I got this best practice from this url https://stackoverflow.com/questions/13171907/best-way-to-enum-nsstring